chcemvediet/static/main/css/navs.sass

```diff
@@ -0,0 +1,28 @@
+
+.chv-nav
```

Comment:
Na mobile chyba vertikalna medzera medzi obsahom a menu:

![mobile](https://user-images.githubusercontent.com/5547275/106395226-58a6bf80-6401-11eb-9696-1ca22658eefc.png)

Vid ako to je na navrhu https://github.com/martinmacko47/chcemvediet/issues/67#issuecomment-170110978 (uplne dole)